BSP/STM32F413/Src/BSP_Fans.c

```diff
@@ -107,10 +119,10 @@ int BSP_Fans_GetSpeed(uint8_t fan){
         return TIM8-> CCR2 / DIVIDER;
         break;
     case 3:
-        return TIM8-> CCR3 / DIVIDER;
+        return TIM12-> CCR1 / DIVIDER;

```

Comment:
TIM_GetCapture4(TIMx) performs the same function but is provided by stm. Should probably replace this with that function instead.

Reply:
Will do